src/main/java/io/kestra/plugin/gcp/gke/ClusterMetadata.java

```diff
@@ -28,35 +28,34 @@
 @Plugin(
     examples = {
         @Example(
-            title = "Fetch a gke cluster metadata",
+            title = "Fetch a GKE cluster's metadata.",
             code = {
-                "name: \"gke-metas\"",
-                "projectId: my-project-id",
-                "zone: eu-west-1c",
+                "clusterProjectId: my-project-id",
+                "clusterZone: eu-west1-c",
```

Comment:
Shouldn't it be `europe-west1-c` instead.

Reply:
Changed it to "us-central1-a" where it definitely works.
![Screenshot 2024-03-19 at 3 45 41 PM](https://github.com/kestra-io/plugin-gcp/assets/6594483/58a58bea-e8f2-45b2-a747-d6ec9d4dac4e)

Reply:
Corrected. I had actually tried it on "us-central1-a", now tried in "europe-west1-c" as well.